src/t_stream.c

```diff
@@ -2880,68 +2899,90 @@ void xpendingCommand(client *c) {
     long long count = 0;
     long long minidle = 0;
     int startex = 0, endex = 0;
+    int j = 3;
+    int mkgroup = 0;   
 
-    /* Start and stop, and the consumer, can be omitted. Also the IDLE modifier. */
-    if (c->argc != 3 && (c->argc < 6 || c->argc > 9)) {
+    if (c->argc > 10) {
         addReplyErrorObject(c,shared.syntaxerr);
         return;
     }
 
-    /* Parse start/end/count arguments ASAP if needed, in order to report
-     * syntax errors before any other error. */
-    if (c->argc >= 6) {
-        int startidx = 3; /* Without IDLE */
-
-        if (!strcasecmp(c->argv[3]->ptr, "IDLE")) {
-            if (getLongLongFromObjectOrReply(c, c->argv[4], &minidle, NULL) == C_ERR)
-                return;
-            if (c->argc < 8) {
-                /* If IDLE was provided we must have at least 'start end count' */
-                addReplyErrorObject(c,shared.syntaxerr);
-                return;
-            }
-            /* Search for rest of arguments after 'IDLE <idle>' */
-            startidx += 2;
-        }
-
-        /* count argument. */
-        if (getLongLongFromObjectOrReply(c,c->argv[startidx+2],&count,NULL) == C_ERR)
+    while(j < c->argc) {
+       if (!strcasecmp(c->argv[j]->ptr, "MKGROUP")) {
+          mkgroup = 1;
+	  j++;
+	  continue;
+       }
+       if (!strcasecmp(c->argv[j]->ptr, "IDLE")) {
+          if (j+1 == c->argc) {
+            addReplyErrorObject(c,shared.syntaxerr);
```

Comment:
indentation by 4, not 2.

Reply:
Check all code styles and update them.  Thanks a lot